lib/stdlib/src/c.erl

```diff
@@ -1080,8 +1080,12 @@ ls_print([H|T], Width, Len) ->
 ls_print([], _, _) ->
     io:nl().
 
-lengths([H|T], L) -> lengths(T, [length(H)|L]);
-lengths([], L)    -> L.
+max_length([H|T], L) when is_atom(H) ->
```

Comment:
How deep can a `deep_list` (http://erlang.org/doc/man/file.html#type-name) be? Is the same depth as an iolist allowed?

Reply:
Any depth, which `file` will handle, but not `c:ls/1`. I better fix that, too.